mautic/mautic
# [UX] Separate user account fields in sections

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13140 



#### Description:
This PR improves the user experience by simplifying data input and retrieval, reducing confusion, and ensuring efficient management of user information, ultimately resulting in increased user satisfaction and ease of use.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Click on the profile picture
3. Click to open Account
4. Look at the new titles

![image](https://github.com/mautic/mautic/assets/116097999/93d64b3a-bbad-4256-a49b-d0320fb3aab1)
